docs/README.md

```diff
@@ -25,7 +25,24 @@ Install these dependencies:
 
 ### MacOS
 
-Under construction
+1. Install [doxygen](http://www.stack.nl/~dimitri/doxygen/download.html).
+2. Make doxygen available on the command line.
+  1. Locate your doxygen executable.  One possible location is
```

Comment:
sure:  

1b. Alternatively, use homebrew to install: `brew install doxygen`. The executable will be installed in `/usr/local/bin` which is already in your path.